src/Core/AD.jl

```diff
@@ -1,3 +1,46 @@
+module AD
+
+import ForwardDiff
+import Flux.Tracker
+using ..VarReplay
```

Comment:
Probably dumb question - what do `..` and `...` do?

Reply:
One dot searches for the module in the same global scope (as opposed to normal package directories). Two dots go up one level, three dots go up 2 levels, etc. For example:

```julia
module A
    module B
    end
    using .B

    module C
         using ..B        
         module D
             using ...B
         end
         using .D
    end
end
```

Reply:
Great thx!